test/run_pass/test_stream_writable.js

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2015-present Samsung Electronics Co., Ltd. and other contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+var assert = require('assert');
+
+var Writable = require('stream').Writable;
+var writable = new Writable({highWaterMark: 0});
+
+var writable2 = Writable({highWaterMark: 0});
+
+assert.throws(function() { writable._write(0, 0, 0); }, Error);
+
+process.platform = 'nuttx';
```

Comment:
Wanted to test if the EOF event runs without errors, making the process believe it's nuttx, so the code runs.